packages/frontend-plugin-api/src/wiring/createPlugin.test.ts

```diff
@@ -150,6 +148,8 @@ describe('createPlugin', () => {
     });
     expect(plugin).toBeDefined();
 
+    expect(plugin.getExtension('test/1')).toBe(Extension1);
```

Comment:
Would it be a good idea to include a test that attempts to get an extension with an invalid id?